src/main/resources/db/migration/postgresql/V1_4__ALTER_TABLE_audit.sql

```diff
@@ -1 +1 @@
-ALTER TABLE audit ALTER COLUMN detail type JSONB USING detail::JSONB;
+ALTER TABLE audit ALTER COLUMN detail type JSONB USING detail::JSONB;
```

Comment:
Can you revert  this file - if it changes at all (even whitespace) then the entire update will be rejected by Flyway.

Reply:
Whitespace reinstated on latest commit.